### ISSUE TITLE
Simple year view (3x4 month grid)

### ISSUE THREAD

**Issue by arshaw** [fullcalendar/fullcalendar#2525]
Originally reported on Google Code with ID 2260

```
Hello,
I use fullcalendar and I think is great but it missing a year view...

So, I suggest implement a Year view like that : http://bootstrap-calendar.azurewebsites.net/

What do you think ?

Best wishes. 

Julien Stalder

```

Reported by `zendre4` on 2014-08-28 07:49:39

Imported with **6** stars.

**Comment by AnnoyingTechnology** [comment]
Hello, 
Has this feature been integrated into fullcalendar ?
Thanks.

**Comment by espen** [comment]
No. You can create a custom view though: http://fullcalendar.io/docs/views/Custom_Views/

**Comment by AnnoyingTechnology** [comment]
Yes, I've read that already. But do you plan on integrating the code quoted above [http://bootstrap-calendar.azurewebsites.net/] in a future release ?

**Comment by espen** [comment]
Adam which has created Fullcalendar has already labeled this issue as Status-Accepted so I assume so.

**Comment by AnnoyingTechnology** [comment]
Thank you for that confirmation.

**Comment by arshaw** [comment]
no plans on transplanting that project's code into FC (would be more complicated that it's worth), but i acknowledge the people want this feature and it should be built at some point

**Comment by grundic** [comment]
It's possible to create any view -- week overview, month or year. Right now it needs some monkey patching and code copy/paste. But I believe code could be written so it would be possible to change only some parameters and achieve required view.

For now you can use this: http://jsbin.com/muruqu/4/edit?output

@arshaw, you created a great project, I see it's usage everywhere, excellent work! I know how it's difficult to support open source project when you have job and/or family. Could you please comment here about project status, your plans for it, which features you plan to implement in this year? It would be nice to make a blog post so people would know what to expect.

Thank you.

**Comment by espen** [comment]
@grundic see http://blog.arshaw.com and https://github.com/fullcalendar/fullcalendar/labels

**Comment by grundic** [comment]
@espen thank you for the links, yeah I know a arshaw's blog, I mean I'd like to see a fresh post about plans for this year :)
Is Status-Accepted3 final? Does it mean that it would be merged/implemented?

**Comment by arshaw** [comment]
@grundic this will help you get an idea about future plans: http://fullcalendar.io/blog/2016/01/the-past-year-and-roadmap-for-q1/

**Comment by grundic** [comment]
@arshaw, thank you!

**Comment by h-2** [comment]
Hey everyone,
I came here through nextcloud/calendar#159 also looking for a year view! The blog post mentioned how this was planned for 2016Q2, but apparently its not done yet -- I know how features tend to delay themselves :upside_down_face:

In any case I just wanted to underline that this feature would make many people happy and I hope it's still on the TODO list!

Thank you

**Comment by espen** [comment]
@h-2 feel free to make a pull request :) Also see https://fullcalendar.io/wiki/Requesting-Features/ regarding voting for issues. So far this issue has only one upvote.

Also @arshaw I think this might be a duplicate of https://github.com/fullcalendar/fullcalendar/issues/1140

**Comment by arshaw** [comment]
@espen the issues seemed like dups to me at first as well, but I then realized #1140 involves displaying each individual day, whereas this request involves display events in a month in aggregate, like [this screenshot](https://storage.googleapis.com/google-code-attachments/fullcalendar/issue-2260/comment-2/calendar.jpg). @h-2 can you please make sure to 👍  the appropriate issue?

**Comment by IwishIcanFLighT** [comment]
Hi! Any update on this issue? Would be awesome to have it on nextcloud/calendar.

**Comment by RuudschMaHinda** [comment]
Asking about updates on this year view. This wish was uttered just today during a team meeting at one of my clients.

**Comment by sunjam** [comment]
@p-try per your mention of creating a pr in Nextcloud Calendar for adding a year view, this is the upstream request in Fullcalendar itself.  Hope it helps.

> Fullcalendar supports custom views, so this does not necessarily need to be integrated there first. @h-2 if you are interested in this feature at short notice, have a look at my fork. I have not opened a pull request because I didn't develop it against the vue branch, but I'm planning to do so.

**Comment by bernd-wechner** [comment]
There's a nice example here:

https://github.com/p-try/fullcalendar-yearview/blob/master/examples/year-view.html

Would be wonderful if one of us could try and patch this into a NextCloud instance and report on success and how to. If I find time I will. Not looking good right now though.

**Comment by arshaw** [comment]
This ticket has drifted away from its original intention: displaying a 3x4 grid of cells, each representing a month, and having events render within these top-level cells. I've made a separate ticket that describes this request better: https://github.com/fullcalendar/fullcalendar/issues/7193

Most people assumed this ticket was for displaying 12 mini-months, a need captured in #1140. FORTUNATELY, THAT FEATURE HAS BEEN IMPLEMENTED IN v6.1.0. Please check it out:

- [blog post](https://fullcalendar.io/blog/2023/01/multimonth-and-more)
- [release on github](https://github.com/fullcalendar/fullcalendar/releases/tag/v6.1.0)
